keras_cv/layers/preprocessing/random_hue.py

```diff
@@ -28,40 +28,45 @@ class RandomHue(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
     hue channel (H) by delta. The image is then converted back to RGB.
 
     Args:
-        factor: Either a tuple of two floats or a single float. `factor` controls the
-            extent to which the image saturation is impacted. `factor` =
-            `0.0`, `0.5` or `1.0` makes this layer perform a no-op operation.
+        factor: A tuple of two floats, a single float or `keras_cv.FactorSampler`.
+            factor` controls the extent to which the image saturation is impacted.
+            `factor`=`0.0`, `0.5` or `1.0` makes this layer perform a no-op operation.
             `factor=0.25` and `factor=0.75` makes the image to have fully opposite
             hue value. Values should be between `0.0` and `1.0`.
             If a tuple is used, a `factor` is sampled
             between the two values for every image augmented.  If a single float is
             used, a value between `0.0` and the passed float is sampled.
             In order to ensure the value is always the same, please pass a tuple with
             two identical floats: `(0.5, 0.5)`.
+        value_range:  the range of values the incoming images will have.
+            Represented as a two number tuple written [low, high].
+            This is typically either `[0, 1]` or `[0, 255]` depending
+            on how your preprocessing pipeline is setup.  Defaults to
+            `[0, 255]`.
```

Comment:
I didn't find the default value in the signature. Did I miss anything?

Reply:
Ah - no, no value

Reply:
updated